Uplift/Views/Headers/HomeScreenHeaderView.swift

```diff
@@ -9,17 +9,18 @@ import SnapKit
 import UIKit
 
 class HomeScreenHeaderView: UIView {
-
+    
```

Comment:
Whitespace; you should turn on the swiftlint and make sure there's no styling related warnings before PR.